packages/controller/src/lib/setup.ts

```diff
@@ -660,6 +677,7 @@ async function processCommand(command, args, params, callback) {
 
                         // we update existing things, in first as well as normnal setup
                         // Rename repositories
+                        // eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
await import maybe?

Reply:
same as above, I would like to, but as it is plain js he does not infer anything and we need an expect error a line below